engine/src/main/java/org/terasology/rendering/world/WorldRendererLwjgl.java

```diff
@@ -442,14 +406,14 @@ public void renderWorld(Camera camera) {
         DefaultRenderingProcess.getInstance().beginRenderLightGeometryStencilPass();
         Material program = Assets.getMaterial("engine:prog.simple");
```

Comment:
I can understand where an asset "engine:simple" is located. What I didn't manage to understand yet, even looking at the Asset Management code, is where is that "prog." substring is handled and why is it necessary.